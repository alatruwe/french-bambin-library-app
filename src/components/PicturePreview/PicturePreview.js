export default function PicturePreview(props) {
  return (
    <>
      <img id="output" width="200" src={props.src} />
      <button onClick={props.delete}>x</button>
    </>
  );
}
