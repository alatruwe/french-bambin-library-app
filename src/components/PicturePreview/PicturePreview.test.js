import React from "react";
import ReactDOM from "react-dom";
import PicturePreview from "./PicturePreview";
import renderer from "react-test-renderer";

describe(`PicturePreview component`, () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<PicturePreview />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer.create(<PicturePreview />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
