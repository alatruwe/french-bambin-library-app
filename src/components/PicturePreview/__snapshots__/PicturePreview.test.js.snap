// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PicturePreview component renders the UI as expected 1`] = `
Array [
  <img
    id="output"
    width="200"
  />,
  <button>
    x
  </button>,
]
`;
